data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,65 @@ mission "Arachi Cartoonist 2"
+			choice
+				`	"Who are you, exactly? How did you even get here?"`
+				`	"What are you doing here in the Coalition?"`
+			`	"Where do I begin? Well, for starters, name's Alitis. Was Athis before, but I took a Saryd name after I got here. Back in the 2800s, the crew I was on found some aliens somewhere far north of Earth. We didn't get much from them though, as they were fighting another group of aliens, ones who had jump drives. We didn't want any part of the fighting because the battle wasn't favorable to us, but we managed to loot a jump drive from the second group of aliens. After that, we started exploring disconnected systems to the east. Eventually, we joined an enclave in the Dirt Belt, but it didn't take long for the Navy to find us there. While evacuating, some of the others wanted to head back east, but I knew better, so I banked on heading further west, and after some time got here."`
+			choice
+				`	"You 'knew better?' What do you mean?"`
+				`	"What, did you have a change of heart once you got here?"`
+			`	He sits on the ground for a while, thinking. "I was one of those that fought humanity after the first Turf War. I was ambushed, cornered many times - fear wasn't a stranger to me. After we stole that jump drive and ventured east, however, someone or... something rather, some voice spoke to me. It was the first time I felt scared. After that, I wasn't inclined to go back east with the others, no matter how silly a reason they thought it was. When I got here, me and my crew helped the Heliarchs with translations for humans, and I told them about humanity and the Alphas. They took me in, but demanded my jump drive, so I handed it over. Damn thing got me in more trouble than it was worth anyway."`
+			choice
+				`	"So where are the other Alphas that came with you?"`
+					goto others
+				`	"What did the voice say?"`
+			`	He forces a chuckle out. "You got here, so you have a jump drive, I assume? Go on, head east and maybe it'll speak to you too. Find out if it classifies you as 'disposable' as well."`
+				goto end
+			label others
+			`	"Others? Oh, you misunderstood me. I took a crew of brigands and pirates. They're all long dead now, most of old age. I'd say they made the right choice in coming with me, no matter how much they protested that the aliens here were too... alien."`
+			label end
+			`	The Heliarchs inform you that the medic has arrived, and bring you further down the hallway to a room where they treat your bruises. Although the Alpha was right in that you weren't injured much, the Heliarchs insist on cleaning up your injuries. Once they're done, an agent escorts you back to your ship.`
```

Comment:
When did he say anything about my injuries?
Perhaps change this up to mention how Alitis looks as you leave. Since you're brought further down the hallway, you probably come back up the hall after you've been patched up. I suggested having Alitis turn away from you after your last question, so perhaps you walk past him still staring at the wall.